chore: Minimal set of permissions to deploy che on openshift (#2794)
codenvy/che-docs: modules/administration-guide/pages/permissions-to-install-che-on-openshift-using-the-web-console.adoc

:_content-type: REFERENCE
:description: Minimum set of permissions required to install {prod-short} on OpenShift using the web console
:keywords: permissions, openshift, installing
:navtitle: Permissions to install {prod-short} on OpenShift using web console

[id="permissions-to-install-che-on-openshift-using-the-web-console"]
= Permissions to install {prod-short} on OpenShift using the web console

> Here you can find the minimal set of permissions required to install {prod-short} on an OpenShift cluster using the web console:

Reviewer: ```suggestion
Below is the minimal set of permissions required to install {prod-short} on an OpenShift cluster using the web console:
```
